Ginger/Ginger/BusinessFlowsLibNew/NewAutomatePage.xaml.cs

```diff
@@ -73,19 +73,13 @@ public NewAutomatePage(BusinessFlow businessFlow)
 
             //WorkSpace.Instance.GingerRunner.CurrentBusinessFlow.PropertyChanged += CurrentBusinessFlow_PropertyChanged;
             //WorkSpace.Instance.GingerRunner.GingerRunnerEvent += GingerRunner_GingerRunnerEvent;
-
-
-            App.PropertyChanged += App_PropertyChanged;
+            
 
```

Comment:
please undo newAutomatePage changes, it is not relevent and will cause later conflict because i'm working on this page